diff --git a/node_modules/react-native-fbads/android/app/build.gradle b/node_modules/react-native-fbads/android/app/build.gradle
index 39e889c..3c67b85 100644
--- a/node_modules/react-native-fbads/android/app/build.gradle
+++ b/node_modules/react-native-fbads/android/app/build.gradle
@@ -26,7 +26,7 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation "com.android.support:recyclerview-v7:${safeExtGet('supportLibVersion', '26.1.0')}"
-    implementation 'com.facebook.android:audience-network-sdk:6.2.+'
-    implementation 'com.facebook.android:facebook-android-sdk:6.+'
+    implementation "com.android.support:recyclerview-v7:28.0.0"
+    implementation 'com.facebook.android:audience-network-sdk:6.16.0'
+    implementation 'com.facebook.android:facebook-android-sdk:17.0.0'
 }
diff --git a/node_modules/react-native-fbads/ios/ReactNativeAdsFacebook/EXAdSettingsManager.m b/node_modules/react-native-fbads/ios/ReactNativeAdsFacebook/EXAdSettingsManager.m
index 56bb991..335f874 100644
--- a/node_modules/react-native-fbads/ios/ReactNativeAdsFacebook/EXAdSettingsManager.m
+++ b/node_modules/react-native-fbads/ios/ReactNativeAdsFacebook/EXAdSettingsManager.m
@@ -7,7 +7,7 @@
 #import <FBAudienceNetwork/FBAudienceNetwork.h>
 #import <AppTrackingTransparency/AppTrackingTransparency.h>
 
-#import <FBSDKCoreKit/FBSDKSettings.h>
+// #import <FBSDKCoreKit/FBSDKSettings.h>
 
 @implementation RCTConvert (EXNativeAdView)
 
@@ -126,7 +126,7 @@ RCT_EXPORT_METHOD(requestTrackingPermission:(RCTPromiseResolveBlock)resolve reje
 
 RCT_EXPORT_METHOD(setAdvertiserIDCollectionEnabled:(BOOL)enabled)
 {
-    [FBSDKSettings setAdvertiserIDCollectionEnabled:enabled];
+    // [FBSDKSettings setAdvertiserIDCollectionEnabled:enabled];
 }
 
 
